### ISSUE TITLE
Ctrl-R Ctrl-S for linux search will freeze tabby

### ISSUE THREAD

**Issue by lawweiliang** [Eugeny/tabby#9731]
**Usage**:
Window Subsystem: WSL-V2
Subsytem Operating System: Debian

**To Reproduce**:
When I type Ctrl-R | Ctrl-S over the tabby terminal, the terminal is freeze.

**Comment by zuilintan** [comment]
Yes, in a normal Linux terminal,  ctrl+s will freeze tabby, too.

**Comment by Clem-Fern** [comment]
Hey, I'm not able to reproduce. Ctrl+Q unfreeze terminal normaly.
Do you have any error in logs ?
